source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md

```diff
@@ -1,53 +1,100 @@
 title: List Upload Troubleshooting
 weight: 0
 layout: page
-zendesk_id: 206317698
 navigation:
   show: true
 ---
 
- **My upload file is not accepted**
+{% info %}
+In the case of an error, we will often send an email to you in the case of a list upload failure. Please check the email from SendGrid that outlines the results of your CSV upload, including a downloadable error report.
 
-**Character Encoding**
+![]({{root_url}}/images/listupload_error.png)
 
-If your contacts list has non-English characters, please make sure that your file is a CSV file with UTF-8 formatting.
+{% endinfo %}
 
-**Text that looks like a number**
+{% anchor h2 %} General Formatting Rules {% endanchor h2 %}
 
-Make sure that you don’t have text fields that look like numbers. This excludes whole numbers (1,2,3,4) or decimals/floats (3.1415). Text that look like numbers are those that contain punctuation characters like a dash, underscore, parenthesis, or multiple dots.
 
-Some examples:
+{% anchor h3 %} Character Encoding {% endanchor h3 %}
 
+If your contacts list has non-English characters, please make sure that you're using a CSV file that is UTF-8 encoded.
+
+
+{% anchor h3 %} Not all addresses populate in UI {% endanchor h3 %}
+
+SendGrid will remove duplicate and invalid email addresses (including email addresses with special characters) from your list during upload, so the number of contacts can potentially change between your CSV and your Marketing Campaigns list.
+
+
+{% anchor h3 %} Text that looks like a number {% endanchor h3 %}
+
+Make sure that you don’t have text fields that look like numbers. Text that look like numbers are those that contain punctuation characters like a dash, underscore, parenthesis, or multiple dots.
+
+```
 - US Zip Code: 80202-1713
 - Phone numbers: “(555) 555-5555” or “555.555.5555”
 - Monetary Values with the currency indicator such as $3.50 or €5.73
+```
+{% anchor h3 %} CSV Headers can be blank {% anchor h3 %}
 
-**Data formatting**
-
-Make sure that your dates are properly formatted in your CSV. If you created your CSV with Excel, you may need to force the date format before exporting to CSV. The date format needs to be one of the following: MM/DD/YYYY, MM/D/YYYY, M/D/YYYY, or M/DD/YYYY
+- Header fields can be blank in your CSV, but must be named and have their type defined later during the upload process.
 
-**Missing Header**
+![]({{root_url}}/images/listupload_5.png)
 
-Your CSV should always have a header row. The rules for this are:
+{% anchor h3 %} Date formatting {% endanchor h3 %}
 
-1. You must have an “email” column.
-2. Capitalization doesn’t matter.
-3. The other columns can be blank.
+Make sure that your dates are properly formatted in your CSV. If you created your CSV with Excel, you may need to force the date format before exporting to CSV. The date format needs to be one of the following: `MM/DD/YYYY, MM/D/YYYY, M/D/YYYY, or M/DD/YYYY`
```

Comment:
Properly format any dates in your CSV. If you created your CSV with Excel and it is not displaying properly, force the date format before exporting to CSV. Use one of the following formats: `MM/DD/YYYY, MM/D/YYYY, M/D/YYYY, or M/DD/YYYY`